query/config_changes.go

```diff
@@ -150,16 +150,20 @@ func (t *CatalogChangesSearchRequest) Validate() error {
 }
 
 type ConfigChangeRow struct {
-	ExternalChangeId string     `gorm:"column:external_change_id" json:"external_change_id"`
-	ID               string     `gorm:"primaryKey;unique_index;not null;column:id" json:"id"`
-	ConfigID         string     `gorm:"column:config_id;default:''" json:"config_id"`
-	ChangeType       string     `gorm:"column:change_type" json:"change_type" faker:"oneof:  RunInstances, diff" `
-	Severity         string     `gorm:"column:severity" json:"severity"  faker:"oneof: critical, high, medium, low, info"`
-	Source           string     `gorm:"column:source" json:"source"`
-	Summary          string     `gorm:"column:summary;default:null" json:"summary,omitempty"`
-	CreatedAt        *time.Time `gorm:"column:created_at" json:"created_at"`
-	ConfigName       string     `gorm:"column:name" json:"name,omitempty"`
-	ConfigType       string     `gorm:"column:type" json:"type,omitempty"`
+	ExternalChangeId  string     `gorm:"column:external_change_id" json:"external_change_id"`
+	ID                string     `gorm:"primaryKey;unique_index;not null;column:id" json:"id"`
+	ConfigID          string     `gorm:"column:config_id;default:''" json:"config_id"`
+	ChangeType        string     `gorm:"column:change_type" json:"change_type" faker:"oneof:  RunInstances, diff"`
+	Severity          string     `gorm:"column:severity" json:"severity"  faker:"oneof: critical, high, medium, low, info"`
+	Source            string     `gorm:"column:source" json:"source"`
+	Summary           string     `gorm:"column:summary;default:null" json:"summary,omitempty"`
+	CreatedAt         *time.Time `gorm:"column:created_at" json:"created_at"`
+	ConfigName        string     `gorm:"column:name" json:"name,omitempty"`
+	ConfigType        string     `gorm:"column:type" json:"type,omitempty"`
+	CreatedBy         *uuid.UUID `gorm:"column:created_by" json:"created_by,omitempty"`
+	ExternalCreatedBy string     `gorm:"column:external_created_by" json:"external_created_by,omitempty"`
+	AuthorName        string     `json:"author_name,omitempty"`
+	AuthorAvatar      string     `json:"author_avatar,omitempty"`
```

Comment:
I don't think we need to include these - The vast majority of changes will not be linked to a person and we can do the lookup and cache of person -> name, avatar client side

Reply:
So just return source, external_created_by and created_by and let frontend decide ?